www/app/hardware/Hardware.html

```diff
@@ -1127,6 +1127,56 @@ <h2><span data-i18n="Hardware">Hardware</span>:</h2>
 			</tr>
 		</table>
 	</div>
+	<div id="divmercedes">
```

Comment:
Agree.

Only challenge is that all (both the Mercedes and Tesla) hardware screens using the eVehicle framework have to put their data into the same constructor of the eVehicle class. And I thinks that all implementation classes used by the eVehicle class should also share a single constructor definition which means that some flexibility has to come from a more flexibel additional parameter in the constructor or custom properties that are set after construction. But that leads to car/brand specific code in the eVehicle class, which I think we should try to avoid.

I can at least make not used parameters hidden so the user is not bothered with them.